code/components/mqtt_ctrl/server_mqtt.cpp

```diff
@@ -854,6 +863,9 @@ static bool publishHADiscoveryTopic(const strHADiscoveryData *_data, const int _
             configurationTopic = cfgDataPtr->homeAssistant.discoveryPrefix + "/sensor/" + nodeID + "/" + topicNameID + "/config";
         }
     }
+    else if (_data->topicName == "reboot") { // Special case: Cycle start
```

Comment:
Please reuse existing code. It seems identical than cycle_start, therefore add it above with OR. Adapt comment accordingly